sirepo/sim_data/opal.py

```diff
@@ -34,6 +34,10 @@ def fixup_old_data(cls, data):
             if cmd._type == "filter":
                 cmd.type = cmd.type.upper()
             elif cmd._type == "particlematterinteraction":
+                if cmd.type not in [
+                    t[1] for t in cls.schema().enum.ParticlematterinteractionType
```

Comment:
I think this should be t[0]. t[1] is the display name and we store the value inside of models.

Reply:
Good catch. Fixed